calculator.rb

```diff
@@ -0,0 +1,64 @@
+# Calculator Project
+
+# Collect user input for first number
+puts "Welcome to calculator! Please enter the first number you'd like to use:"
+num_one = gets.chomp
+# Checks to see if user input is a number. Alerts user of incorrect input
+if num_one.to_i.to_s == num_one
+  num_one = num_one.to_f
+else
+  puts "\nYou did not enter a number. Try again."
+  exit
+end
+
+# Checks to see if user input is a number. Collect user input for second number
+puts "\nPlease enter the second number you'd like to use:"
+num_two = gets.chomp
+# Alerts user of incorrect input
+if num_two.to_i.to_s == num_two
+  num_two = num_two.to_f
+else
+  puts "\nYou did not enter a number. Try again."
+  exit
+end
+
+# Collect user input for operation
+puts "\nPlease enter an operation:"
+operation = gets.chomp
+
```

Comment:
A good way to reduce duplication here would be to assign your output message to a variable. For example:

    message = "\nHere are the calculations: "

    if operation == "addition" || operation == "+"
        answer = num_one + num_two
        operator = "+"
    # rest of block here ...
    end

    puts message + "#{num_one} #{operator} #{num_two} = #{answer}"

This way if you want to change your message, you only have to do it in one place. Plus it saves typing :)